scripts/autotools/libmongoc/CheckSSL.m4

```diff
@@ -1,28 +1,30 @@
 PHP_ARG_WITH([mongodb-ssl],
              [whether to enable crypto and TLS],
-             [AS_HELP_STRING([--with-mongodb-ssl=@<:@auto/no/openssl/libressl/darwin@:>@],
+             [AS_HELP_STRING([--with-mongodb-ssl=@<:@auto/openssl/libressl/darwin/no@:>@],
                              [MongoDB: Enable TLS connections and SCRAM-SHA-1 authentication [default=auto]])],
              [auto],
              [no])
 
-PHP_ARG_WITH([openssl-dir],
```

Comment:
Per [this comment in PHPC-2309](https://jira.mongodb.org/browse/PHPC-2309?focusedCommentId=5828671&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-5828671), I'm leaning towards reverting this change and leaving `--with-openssl-dir` in place until 2.0 when we remove _all_ deprecated `configure` options. This is very low maintenance at this point, and I wasn't confident enough to remove the entire `else` condition for `PKG_CHECK_MODULES` that falls back to `PHP_CHECK_LIBRARY` since we still use that for non-TLS purposes (e.g. compression and SASL checks fall back in the same manner).

Reply:
Keeping that logic in until 2.0 sounds fine to me 👍